### DISCUSSION TITLE
Conserver les classes achetées

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#599] [Idées acceptées / déjà implémentées]
Lorsque l'on achète une classe on devrait la garder, comme ça quand on en achète une autre on peut toujours reprendre l'ancienne classe, avec un cooldown d'une semaine bien sûr pour que le joueur ne puisse pas prendre tank pour les rapports et ensuite une autre classe pour les combats.

**Comment by Bombafox** [comment]
Je comprends pas pourquoi c'est toujours pas dispo ça, surtout au prix des classes après le niv 80.
Surtout que du coup t'es obligé de payer pour tester pour ensuite repayer pour reprendre l'ancienne.
Désolé mais moi je m'appelle pas LePourfendeur. 🗿

**Comment by BananePlantain** [comment]
On signe où ?

**Comment by romain22222** [comment]
BON DRAFT, tu la passes en accepté oui ou merde ?

**Reply by BastLast** [reply]
Idk tu le dev ?

**Reply by romain22222** [reply]
quand j'aurais le temps, si personne le fait

**Reply by BastLast** [reply]
> quand j'aurais le temps, si personne le fait
> 

Bon bah en attendant ça reste là

**Comment by BastLast** [comment]
Suivre le travail sur cette proposition via ce lien : #1884
